### ISSUE TITLE
Add sample configs for doom.el

### ISSUE THREAD

**Issue by Trevoke** [Trevoke/org-gtd.el#78]
e.g. 

```elisp
;;; Org-gtd
(use-package! org-gtd
  :after org
  :config
  (org-edna-mode)
  (setq org-gtd-directory "~/documents/org/gtd")
  (setq org-gtd-default-file-name "actionable")
  (setq org-edna-use-inheritance t)
  (map!
   :leader
   (:prefix-map ("d" . "org-gtd")
    :desc "Capture"        "c"  #'org-gtd-capture
    :desc "Engage"         "e"  #'org-gtd-engage
    :desc "Process inbox"  "p"  #'org-gtd-process-inbox
    :desc "Show all next"  "n"  #'org-gtd-show-all-next
    :desc "Stuck projects" "s"  #'org-gtd-show-stuck-projects))

  (map!
   :map org-gtd-process-map
   :desc "Choose"         "C-c C-c" #'org-gtd-choose))
```

**Comment by jlcolbert** [comment]
I have a small suggestion for the Doom config. In some (out of the way, on Discourse) documentation, it cautions against using `:prefix-map`. Apparently it has something to do with how it wraps around something else.

Here's mine:

```
(use-package! org-gtd
  :after org
  :config
  (org-edna-mode)
  (setq org-edna-use-inheritance t)
  (map! :leader
        (:prefix ("d" . "org-gtd")
         :desc "Capture"        "c"  #'org-gtd-capture
         :desc "Engage"         "e"  #'org-gtd-engage
         :desc "Process inbox"  "p"  #'org-gtd-process-inbox
         :desc "Show all next"  "n"  #'org-gtd-show-all-next
         :desc "Stuck projects" "s"  #'org-gtd-show-stuck-projects))
  (map! :map org-gtd-process-map
        :desc "Choose" "C-c c" #'org-gtd-choose))
```

**Comment by Trevoke** [comment]
Fixed by #132
